ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -115,21 +127,10 @@ export const getRegistrationTasks = () => serverConfigStore.getValue() ? ([
     },
     onClick: redirectToTraining
   }, {
-    key: 'eraCommons',
-    completionPropsKey: 'eraCommonsLinked',
-    title: 'Login to eRA Commons',
-    description: 'Link to your eRA Commons account to the workbench to gain full access to data and tools.',
-    buttonText: 'Login',
-    completedText: 'Linked',
-    completionTimestamp: (profile: Profile) => {
-      return profile.eraCommonsCompletionTime || profile.eraCommonsBypassTime;
-    },
-    onClick: redirectToNiH
-  }, {
-    key: 'dataUseAgreement',
+    key: 'dataUserCodeOfConduct',
     completionPropsKey: 'dataUseAgreementCompleted',
```

Comment:
Looking good!  I'd like one last change: update this line so it doesn't refer to a "data use agreement"